Compendium/WWN/Creating-Adventures/Creating-Combat-Challenges.md

```diff
@@ -1,6 +1,195 @@
+
+Take caution to not force combat challenges. PCs are often resourceful enough to dodge combat; if there's a will, there's a way. Some particularly charismatic groups may find a way to force their peaceful solution on the challenge. It's often in the PCs best interest to avoid combat if it isn't absolutely necessary; as the GM try not force your party into unavoidable combat encounters. Combat is appropriate when the PCs have elected to slay a dragon, but it is not appropriate if the party decides they're just after the dragon's gold and you decide to orchestrate things such that the only way to steal the dragon's hoard is to destroy it in battle.
+
+The purpose of this section in creating a combat challenge is to arm the GM with all the facts needed to run the encounter. For lesser challenges, like determining the disposition of an [[Anak]] with a treat who happens to be lurking nearby, you may only need a brief line detailing its combat statistics. Greater challenges, like trying to kill a cult leader, you may need something more substantive.
+
+### The Essentials
+
+The first step in fleshing out a combat challenge is to map out a list of the associated creatures and NPCs. Once you have determined who's present, generate or note their combat statistics. Consider the following list:
+
+1. Think about the target
+2. Think about any surrounding threats, associates, or minions that it might have
+    - Merchants have guards, cult leaders have underling fanatics, and Anak warriors often roam with their warbands
+    - Allied groups may have individuals in the area
+    - The target may have employed or forced others into accomplishing their aims
+    - Populating a [[Deep]] or some other highly-populated location of monsters, the creatures likely belong in that environment, or have adapted to live in the area
+3. Avoid thinking about "balancing" combat challenges
+    - Don't adjust enemy numbers to make the encounter easier or harder based on the power level of the party; rather, tell the narrative in a logical and coherent way such that the PCs can engage with it
+    - i.e. The King's Guard are ferocious, the Anak tribe has scores of combat-capable warriors, the evil sorcerer is surrounding by their inhuman minions, and the most infamously dangerous swordsman in the country regardless of whether your party is level one or level ten
+4. Think about the context of your adventure's purpose
+   - A [[Deep]] for novice heroes should contain winnable combat challenges
+   - There should be a logical reason for why the [[Deep]] is filled with PC-slaughtering hell-beasts
+   - Have trivial and grand foes
+   - Refer to [[page 295]] for aid in assessing a combat challenge's likelihood of killing PCs
```

Comment:
is there a way to query a file with the page 295 metadata?

Reply:
Not off the top of my head. What I plan to do is create a direct link to the page in question but leave the alias as [[path-to-page|page 295]].